tests/unit/processing_container/test_process.py

```diff
@@ -56,9 +56,10 @@ def test_make_seurat(self):
     def test_make_loom(self):
         make_loom(str(self.real_h5ad_filename))
 
-    # TODO: Remove mocking of validate_h5ad_file after validation process is updated to be 2.0 compliant.
-    @patch("backend.corpora.dataset_processing.process.validate_h5ad_file")
+    # TODO: Remove mocking of validate_h5ad_file after validation process is updated to be 2.0 compliant.`
```

Comment:
is it time to address this TODO? Seems like we should also verify that labels were added to the newly written file. The original test only appears to be ensuring that `process()` runs w/o raising an error, but we could add some assertions here I think.  Unless this is level of testing is being handled elsewhere?

Reply:
Additional testing won't be part of this ticket, as per ticket comments (see https://app.zenhub.com/workspaces/single-cell-5e2a191dad828d52cc78b028/issues/chanzuckerberg/single-cell-data-portal/1297#issuecomment-914721648). That said, I agree this should be done on a follow up ticket. It will be tricky since with the current test, we're downloading an existing h5ad file in a staging dataset (therefore, with labels), while we should change it to use a "raw" h5ad (without labels). That said, downloading it from a Dropbox folder instead of using S3 should be easy enough and will also make the test independent of the staging environment.

Reply:
Great, maybe just make that ticket soon (with your above insights) so we don't forget. :)